posix.mak

```diff
@@ -55,7 +55,8 @@ STABLE_RDMD=$(STABLE_DMD_ROOT)/dmd2/$(OS)/$(if $(filter $(OS),osx),bin,bin$(MODE
 MOD_EXCLUDES_PRERELEASE=$(addprefix --ex=, gc. rt. core.internal. core.stdc.config core.sys.	\
 	std.c. std.algorithm.internal std.internal. std.regex.internal. 			\
 	std.windows.iunknown std.windows.registry etc.linux.memoryerror std.stream std.cstream	\
-	std.socketstream std.experimental.ndslice.internal)
+	std.socketstream std.experimental.ndslice.internal \
+	tk. msvc_dmc msvc_lib)
```

Comment:
We should probably add more DMD files to this list (but this can be done after this PR)